packages/charts/src/components/CarbonChart/CarbonChart.tsx

```diff
@@ -460,6 +476,14 @@ export const CarbonChart = React.forwardRef<HTMLDivElement, CarbonChartProps>(
       line: LineChart,
       lollipop: LollipopChart,
       pie: PieChart,
+      radar: RadarChart,
+      boxplot: BoxplotChart,
+      bubble: BubbleChart,
+      bullet: BulletChart,
+      gauge: GaugeChart,
+      histogram: HistogramChart,
+      meter: MeterChart,
+      scatter: ScatterChart,
```

Comment:
We will need to go through all of these charts and update the custom styles. @orion-cengage will need to review them and provide feedback.

Reply:
We could also merge as is (after the release) and use the separate tickets to address the styling.